test/examples/gridworld.jl

```diff
@@ -7,7 +7,7 @@ end
     env = GW.GridWorld()
     reset!(env)
     while !terminated(env)
-        act!(env, rand(actions(env)))
+        act!(env, actions(env)[rand(1:length(actions(env)))])
```

Comment:
I think it's better to skip this test on version 1.0 rather than make this modification. It is much harder to read and not what we want people to do (because `actions(env)` is more likely to have `rand` than `getindex`